contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -106,6 +115,18 @@ contract TrueCreditAgency is UpgradeableClaimable {
         return min(creditAdjustmentCoefficient.mul(MAX_CREDIT_SCORE - score).div(score), 50000);
     }
 
+    function utilizationAdjustmentRate(ITrueFiPool2 pool) public view returns (uint256) {
+        uint256 liquidRatio = pool.liquidRatio();
+        if (liquidRatio == 0) {
+            // if utilization is at 100 %
+            return type(uint256).max;
```

Comment:
Return 50000 here same as for credit scrore adjustment and extract to MAX_RATE_CAP constant